pkg/policymanager/manager.go

```diff
@@ -228,25 +238,31 @@ func (p Policy) IsDirect() bool {
 }
 
 func (p Policy) IsAttachedTo(objRef common.GKNN) bool {
-	if p.TargetRef.Kind == "Namespace" && p.TargetRef.Name == "" {
-		p.TargetRef.Name = "default"
-	}
-	if objRef.Kind == "Namespace" && objRef.Name == "" {
-		objRef.Name = "default"
-	}
-	if p.TargetRef.Kind != "Namespace" && p.TargetRef.Namespace == "" {
-		p.TargetRef.Namespace = "default"
-	}
-	if objRef.Kind != "Namespace" && objRef.Namespace == "" {
-		objRef.Namespace = "default"
+	for _, targetRef := range p.TargetRefs {
+		if targetRef.Kind == "Namespace" && targetRef.Name == "" {
+			targetRef.Name = "default"
+		}
+		if objRef.Kind == "Namespace" && objRef.Name == "" {
+			objRef.Name = "default"
+		}
+		if targetRef.Kind != "Namespace" && targetRef.Namespace == "" {
+			targetRef.Namespace = "default"
```

Comment:
I know this refers to a preexisting issue and therefore probably out of scope, but I'm surprised to see targetRef's namespaces defaulting to the `default` namespace. This is counterintuitive in the context of policies. Rather, I'd expect a missing target namespace to default to the namespace of the policy object.